### PR TITLE
#5278 Fix large image overlapping on the pgp_block iframe

### LINKED ISSUE #5278: maximum width of online images to be the full width of the frame?
When I enter an image that is super wide, say 2000px, it will make the encrypted iframe scrollable horizontally, which makes text hard to read.

Most email clients also do this, but I think it's nonsense. I think the image size when rendered should be capped at 100% of available space, meaning it should get shrunk, so that whole image and text remains normally visible without scrolling horizontally.

### PR DESCRIPTION
**Pull request by martgil**
This PR fixes large image overlapping on the pgp_block iframe.

close #5278 

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by martgil** [comment]
Ready for review. Thank you!

File: extension/css/cryptup.css [hunk 1]
```diff
@@ -1482,6 +1482,10 @@ td {
   margin-top: 8px;
 }
 
+#pgp_block img {
+  width: 100%;
```
Comment (sosnovsky) [inline extension/css/cryptup.css:1486]: I think it should be `max-width` property, as currently even small images are stretched to 100% width.

Comment (martgil) [reply extension/css/cryptup.css]: Ah, yes. I missed this one. I have corrected it by using `max-width`. Thanks!

File: test/source/browser/controllable.ts [hunk 2]
```diff
@@ -420,8 +420,11 @@ abstract class ControllableBase {
 
   public checkIfImageIsDisplayedCorrectly = async (selector: string) => {
     const isImageDisplayedCorrectly = await this.target.evaluate(selector => {
+      const pgpBlock = document.querySelector('#pgp_block') as HTMLDivElement;
       const img = document.querySelector(selector) as HTMLImageElement;
-      return img.naturalWidth !== 0 && img.naturalHeight !== 0;
+      const imgWidth = img.offsetHeight;
+      const pgpBlockWidth = pgpBlock.offsetHeight;
```
Comment (sosnovsky) [inline test/source/browser/controllable.ts:426]: It should be `img.offsetWidth` and `pgpBlock.offsetWidth`, not `offsetHeight`

Comment (martgil) [reply test/source/browser/controllable.ts]: Sorry, my bad. I unconsciously use `offsetHeight` where I need to check `offsetWidth`.

**Comment by martgil** [comment]
@sosnovsky apology for the back and forth. It's now correctly tested and working well.

**Review by sosnovsky — APPROVED** [review]
Works well 👍

**Pull request merged**